Gemfile

```diff
@@ -1,5 +1,8 @@
 source 'https://rubygems.org'
 
+ruby RUBY_VERSION
+
+gem 'bundler'
```

Comment:
Just so we talk about the same things, some basics: The `Gemfile` for a Rubygem is only a development and testing configuration tool. It won't affect users of the Rubygem.

The `ruby RUBY_VERSION` trick tells Bundler to "narrow your installation requirements to gems that support _our current_ Ruby". Without this annotation, Bundler only has the version requirements like `gem 'example', '~> 2.2', '>= 2.2.2'` to narrow its gem version resolution with.

This allows a jruby-1.7.27 (Ruby 1.9) to navigate dependency chains which have Ruby 1.9/2.x differences (like json or mime-types libraries) in their "gem upgrade path", picking 1.9-compatible gems along the way.

---

_Vacillating_: Since a build takes a long time, I'd be happy to make the experimentation with dropping `gem 'bundler'` outside the context of this PR.